tests/yam/agama/auto/auto.pm

```diff
@@ -0,0 +1,23 @@
+## Copyright 2023 SUSE LLC
```

Comment:
probably we just need `tests/yam/agama/auto.pm` for now or we plan to do more version of this, I guess this would represent some sort o general mechanism, but I would not discard it wouldn't the only one, up to you.